web/src/components/forms/blank-project-form.tsx

```diff
@@ -129,9 +130,16 @@ sample_table: samples.csv
           placeholder="name"
         />
         <span className="mx-1 mb-1">:</span>
-        <input {...register('tag')} id="blank_tag" type="text" className="form-control" placeholder="default" />
+        <input {...register('tag', {
+          required: false,
+            pattern: {
+              value: /^[a-zA-Z0-9_-]+$/,
+              message: "Project Tag must contain only alphanumeric characters, '-', or '_'.",
+            },
+        })} id="blank_tag" type="text" className="form-control" placeholder="default" />
       </span>
-      <ErrorMessage errors={errors} name="project_name" render={({ message }) => <p>{message}</p>} />
+      <ErrorMessage errors={errors} name="project_name" render={({ message }) => message ? (<p className='text-secondary pt-1' style={{fontSize: '.75em'}}>{message}</p>) : null} />
+      <ErrorMessage errors={errors} name="tag" render={({ message }) => message ? (<p className='text-secondary pt-1' style={{fontSize: '.75em'}}>{message}</p>) : null} />
```

Comment:
Feel like I've tried to use this before with mixed results. Have you tried?

Reply:
Just tried it myself and it looks great! What do you think about making the error message text red? (`danger`)